server/src/graql/gremlin/spanningtree/graph/DenseWeightedGraph.java

```diff
@@ -35,50 +35,45 @@
 import static com.google.common.collect.Range.closedOpen;
 
 /**
- * @param <V> the type of the nodes stored
+ * Used for testing
```

Comment:
I don't have a context, but why do we have a full class that's only used for tests (according to your comment) in a folder that is not dedicated to tests? 
It was already in here but it might be worth considering to move it in the appropriate test folder

Reply:
Was wondering about this! Implemented in QP refactor 2